lesson03/assignment/index.js

```diff
@@ -0,0 +1,137 @@
+
+                    //Company
+                    const detailsP1 = document.createElement('p');
+                    detailsEl.appendChild(detailsP1);
+
+                    const p1Strong = document.createElement('strong');
+                    p1Strong.innerHTML = `Company: `;
+                    detailsP1.appendChild(p1Strong);
+
+                    const companyNode = document.createTextNode(`${el.company}`);
+                    detailsP1.appendChild(companyNode);
+
+                    //Experience
+                    const detailsP2 = document.createElement('p');
+                    detailsEl.appendChild(detailsP2);
+
+                    const p2Strong = document.createElement('strong');
+                    p2Strong.innerHTML = `Experience: `;
+                    detailsP2.appendChild(p2Strong);
+
+                    const experienceNode = document.createTextNode(`${el.experience}`);
+                    detailsP2.appendChild(experienceNode);
+
+                    // School
+                    const detailsP3 = document.createElement('p');
+                    detailsEl.appendChild(detailsP3);
+
+                    const p3Strong = document.createElement('strong');
+                    p3Strong.innerHTML = `School: `;
+                    detailsP3.appendChild(p3Strong);
+
+                    const schoolNode = document.createTextNode(`${el.school}`);
+                    detailsP3.appendChild(schoolNode);
+
+                    //Major
+                    const detailsP4 = document.createElement('p');
+                    detailsEl.appendChild(detailsP4);
+
+                    const p4Strong = document.createElement('strong');
+                    p4Strong.innerHTML = `Major: `;
+                    detailsP4.appendChild(p4Strong);
+
+                    const majorNode = document.createTextNode(`${el.major}`);
+                    detailsP4.appendChild(majorNode);
+
+                    //Email
+                    const detailsP5 = document.createElement('p');
+                    detailsEl.appendChild(detailsP5);
+
+                    const p5Strong = document.createElement('strong');
+                    p5Strong.innerHTML = `Email: `;
+                    detailsP5.appendChild(p5Strong);
+
+                    //Email link
+                    const emailLink = document.createElement('a');
+                    emailLink.setAttribute('href', 'mailto: paolo@example.com');
+                    emailLink.innerHTML = `${el.email}`;
+                    detailsP5.appendChild(emailLink);
+
+                    //Linkedin
+                    const linkedinContainer = document.createElement('div');
+                    linkedinContainer.setAttribute('class', 'linkedin-container');
+                    detailsEl.appendChild(linkedinContainer);
+
+                        const linkedinLink = document.createElement('a');
+                        linkedinLink.setAttribute('href', 'paolo.linkedinprofile.com');
```

Comment:
Looks like you still may need to convert this from a hard-coded string to the variable that stores the value from the data.json file.  You will also want to add https:// before the link itself so it doesn't try and go to http://localhost:3000/paolo.linkedinprofile.com.